server/server.go

```diff
@@ -151,7 +158,7 @@ func (server HTTPServer) listOfClustersForOrganization(writer http.ResponseWrite
 	}
 }
 
-func (server HTTPServer) readReportForCluster(writer http.ResponseWriter, request *http.Request) {
+func (server HTTPServer) ReadReportForCluster(writer http.ResponseWriter, request *http.Request) {
```

Comment:
this need not be exported (ie. publicly visible)

Reply:
and `goling` basically said the same on CI :)

Reply:
Oh, sorry. I pushed it by mistake. Initially I was using a different (and not productive at all) approach for the UT and I tried to use this function directly, so I needed to change the visibility and forget to get to its correct visibility